### PR TITLE
chore(website-hub): Generate Hub from repo data

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Adds a basic skeleton site for the Hub

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: plugins/source/aws/docs/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# AWS Provider
+# AWS Plugin
 
 The CloudQuery AWS provider extracts and transforms your AWS cloud assets configuration into PostgreSQL.
 
 ## Install
 
-```shell
```
Comment (erezrokah) [inline plugins/source/aws/docs/index.md:7]: Renamed to `bash` as `shell` is not supported by Nextra: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js

File: plugins/source/aws/docs/index.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ export AWS_PROFILE=myprofile
 
 or, configure your desired profile in the `local_profile` field of your CloudQuery `cloudquery.yml`:
 
-```yml title="cloudquery.yml"
```
Comment (erezrokah) [inline plugins/source/aws/docs/index.md:74]: Renamed to `yaml` as `yml` is not supported by Nextra: https://github.com/FormidableLabs/prism-react-renderer/blob/4d8019490a3ad89c310d877a9ae3130084261bc9/src/vendor/prism/includeLangs.js

File: plugins/source/aws/docs/index.md [hunk 3]
```diff
@@ -277,7 +277,7 @@ In order to assume role with MFA, you need to request temporary credentials usin
 aws sts get-session-token --serial-number <YOUR_MFA_SERIAL_NUMBER> --token-code <YOUR_MFA_TOKEN_CODE> --duration-seconds 3600
 ```
 
-export the temporary credentials to your environment variables.
```
Comment (erezrokah) [inline plugins/source/aws/docs/index.md:280]: The MDX parses things `export` is JS code 🤦 . This is a workaround

File: website/components/hub/Plugins.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import type { Plugins as PluginsDetails } from "../../content/hub";
+
+export const Plugins = ({ plugins }: { plugins: PluginsDetails }) => {
```
Comment (erezrokah) [inline website/components/hub/Plugins.tsx:3]: Stub component for the plugins overview, supposed to replace https://hub.cloudquery.io/providers

File: website/scripts/generate-hub.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
+const path = require("path");
+const { promises: fs } = require("fs");
+const Mustache = require("mustache");
+
+Mustache.escape = (text) => text;
+
+const PLUGINS_DATA = {
```
Comment (erezrokah) [inline website/scripts/generate-hub.js:7]: Not sure if there's a better way to map the names

**Comment by dj-stormtrooper** [comment]
Nextra feature request as a follow up to our discussion https://github.com/shuding/nextra/issues/709

**Comment by erezrokah** [comment]
Replaced by https://github.com/cloudquery/cloudquery/pull/1497

**Pull request closed without merging**